apps/native/app/src/screens/home/home.tsx

```diff
@@ -114,12 +117,16 @@ const { useNavigationOptions, getNavigationOptions } =
 export const HomeScreen: NavigationFunctionComponent = ({ componentId }) => {
   useNavigationOptions(componentId)
 
-  useAndroidNotificationPermission()
+  const userProfile = useGetProfileQuery({
+    fetchPolicy: 'cache-first',
```

Comment:
@eirikurn
I’m not sure if cache-first is the right choice here. It will cut down on requests, but if we already have something in the cache, we won’t hit the server at all on app startup. That means things like userProfileLocale and documentNotifications could be out of date.

What do you think? Should we accept one request to the server on startup using cache-and-network, or stick with cache-first to avoid extra calls?